commit a7acee89d68473183cc5021d952a56cdf0ae27d3
Author: Hans Wennborg <hans@chromium.org>
Date:   Mon Oct 19 14:38:53 2020 +0200

    Fix compiler-rt build on Windows after D89640
    
    It moved the cmake code from the else-branch to the if-branch
    but forgot to flip the condition.

diff --git a/compiler-rt/cmake/base-config-ix.cmake b/compiler-rt/cmake/base-config-ix.cmake
index 378749ac219..b273acd1816 100644
--- a/compiler-rt/cmake/base-config-ix.cmake
+++ b/compiler-rt/cmake/base-config-ix.cmake
@@ -169,7 +169,7 @@ macro(test_targets)
     if(COMPILER_RT_DEFAULT_TARGET_ONLY)
       add_default_target_arch(${COMPILER_RT_DEFAULT_TARGET_ARCH})
     elseif("${COMPILER_RT_DEFAULT_TARGET_ARCH}" MATCHES "i[2-6]86|x86|amd64")
-      if(NOT MSVC)
+      if(MSVC)
         if (CMAKE_SIZEOF_VOID_P EQUAL 4)
           test_target_arch(i386 "" "")
         else()
